moto/acm/models.py

```diff
@@ -324,13 +325,15 @@ def validate_chain(self):
             )
 
     def check(self):
-        # Basically, if the certificate is pending, and then checked again after 1 min
-        # It will appear as if its been validated
+        # Basically, if the certificate is pending, and then checked again after a
+        # while, it will appear as if its been validated. The default wait time is 60
+        # seconds but you can set an environment to change it.
+        wait = os.environ.get("MOTO_ACM_VALIDATION_WAIT", 60)
```

Comment:
This should be moved into `moto/settings.py`, where all our env variables are stored